modules/ROOT/pages/index.adoc

```diff
@@ -1,21 +1,17 @@
 [id="openshift-pipelines_{context}"]
 = OpenShift Pipelines
 
-[IMPORTANT]
-====
-OpenShift Pipelines project is based on the upstream project link:https://github.com/tektoncd[Tekton] and is available as a Developer Preview release. Developer Preview releases have features and functionality that might not be fully tested. Customers are encouraged to use and provide feedback on Developer Preview releases. Red Hat does not commit to fixing any reported issues and the provided features may not be available in future releases.
+//:FeatureName: OpenShift Pipelines
+//include::modules/technology-preview.adoc[leveloffset=+1]
```

Comment:
Have removed the Developer Preview note. I have added the technology Preview file, but have commented it out for the time being.